lib/iris/tests/integration/netcdf/test_general.py

```diff
@@ -484,5 +484,43 @@ def test_path_string_save_same(self):
         self.assertCDL(tempfile_frompath)
 
 
+@tests.skip_data
+class TestWarningRepeats(tests.IrisTest):
+    def test_datum_once(self):
+        fnames = [
+            "false_east_north_merc.nc",
+            "non_unit_scale_factor_merc.nc",
+            "toa_brightness_temperature.nc",
```

Comment:
So I've run some tests and I think this file would actually cause a warning repeat if there were files after it, for the sake of making it clear which files are expected not to cause repeats, I think this should be commented out for the time being with a comment noting that this file still causes warning repeats.